tt_test.go

```diff
@@ -0,0 +1,672 @@
+package tupleconv_test
```

Comment:
Let's also add an example, or an integration test with a `go-connector` + `Tarantool` to make sure that all work fine in action.

Reply:
I added this as an example:
https://github.com/tarantool/go-tupleconv/blob/bb5b51eac2f0fd7ba7f6906441a5d7fff92fb2c0/example_test.go#L190